models/match_detail.py

```diff
@@ -33,6 +33,7 @@ def __init__(self, info):
         self.participantId = info.get('participantId')
         self.riotIdGameName = info.get('riotIdGameName')
         self.puuid = info.get('puuid')
+        self.win_color = 'gold' if self.win == 'Victory' else 'red'
```

Comment:
From an architectural point of view, why would a match determine what color it displayed? If we pass "victory" to the front end, can't it use that to pick a color?

Reply:
I'll see.